website/content/en/preview/upgrade-guide.md

```diff
@@ -51,12 +51,12 @@ If you get the error `invalid ownership metadata; label validation error:` while
 In general, you can reapply the CRDs in the `crds` directory of the Karpenter helm chart:
 
 ```shell
-kubectl replace -f https://raw.githubusercontent.com/aws/karpenter{{< githubRelRef >}}pkg/apis/crds/karpenter.sh_provisioners.yaml
-
-kubectl replace -f https://raw.githubusercontent.com/aws/karpenter{{< githubRelRef >}}pkg/apis/crds/karpenter.k8s.aws_awsnodetemplates.yaml
+kubectl apply -f https://raw.githubusercontent.com/aws/karpenter{{< githubRelRef >}}pkg/apis/crds/karpenter.sh_provisioners.yaml
```

Comment:
Does applying work for those who had the CRD previously? I thought you have to replace in this case.

Reply:
No, the difference is similar to the Update/Patch case with client-go (see the docs here: https://blog.atomist.com/kubernetes-apply-replace-patch/). Apply gets information on the current state and then determines the patch that needs to happen on the resource and applies it. It is more declarative vs. imperative since it says "this is what my resource should look like" vs. Create or Update this resource and if it already exists (in the case of create) or if it doesn't exist (in the case of update) it fails

Reply:
Thanks for the explanation!